src/components/upload/FileUpload.js

```diff
@@ -40,40 +48,72 @@ const FileUpload = () => {
     const [projectName, setProjectName] = useState("");
+    const [open,setOpen] = React.useState(false);
+    const handleClose = () => {
+        setOpen(false);
+      };
 
-    const onSubmit = () => {
+      const onSubmit = () => {
+        setOpen(true);
+        uploadProjectFile(projectName, projectFile)
+      };
+    /*const onSubmit = () => {
         setState("uploading");
         uploadProjectFile(projectName, projectFile)
             .then(() => {
                 Notifications.notify("Project created.", "success")
                 setState("idle");
+                setOpen(true);
+                return (
+                    <Dialog
+                    open={open}
+                    onClose={handleClose}>
+                     <DialogTitle> Edit projects </DialogTitle>
+                        <DialogContent>
+                            <DialogContentText>
+                                 <EditorPage></EditorPage>
+                            </DialogContentText>
+                        </DialogContent>
+               </Dialog>
+                );
             })
             .catch((err) => {
                 Notifications.notify(`Failed to create project.\n${err}`, "error")
                 setState("idle");
             })
-    };
+    };*/
 
     return (
         <FileUploadRoot>
             <FileDropZone
                 file={projectFile}
                 onFile={setProjectFile}
             />
-
+                
             <Confirmation>
-                <Input
+               <Box  component="form"  sx={{ '& > :not(style)': { m: 1, width: '25ch' },  }}
+                     noValidate autoComplete="off" >
+                <ProjectName
                     placeholder="Project name"
                     value={projectName}
-                    onChange={(e) => setProjectName(e.target.value)}
-                />
-
-                <UploadButton
-                    onClick={onSubmit}
-                    disabled={!projectName || !projectFile || state === "uploading"}
-                >
-                    Upload file
-                </UploadButton>
+                    onChange={(e) => { setProjectName(e.target.value)}}>
+                    <TextField id="outlined-basic" label="Outlined" variant="outlined" />
+                </ProjectName>
+                <Button variant="contained" 
+                        onClick={onSubmit}  endIcon={<UploadIcon />}>Upload file
+                        
+                </Button>
+                <Dialog
+                    open={open}
+                    onClose={handleClose}>
+                     <DialogTitle> Edit projects </DialogTitle>
+                        <DialogContent>
+                            <DialogContentText>
+                                 <EditorPage></EditorPage>
+                            </DialogContentText>
+                        </DialogContent>
+               </Dialog>
+                </Box>
```

Comment:
```suggestion
                <Dialog open={open} onClose={handleClose}>
                     <DialogTitle>Edit projects</DialogTitle>
                     <DialogContent>
                         <EditorPage/>
                    </DialogContent>
                </Dialog>
            </Box>
```
1. Indentation is wrong
2. `DialogContentText` has no effect
3. `EditorPage` should not be empty tag but rather a single tag